src/index.html

```diff
@@ -16,17 +16,35 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      action="https://github.com/captain-cactus-ua/layout_search-bar-airbnb/settings#dialog-show-repo-delete-menu-dialog"
+      class="container container--shadow"
+    >
+      <div class="search__lens"></div>
+      <input
+        class="search__bar search__bar--placeholder search__bar--weight:focus"
+        id="data-qa-big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="https://github.com/captain-cactus-ua/layout_search-bar-airbnb/settings#dialog-show-repo-delete-menu-dialog"
+      class="container container--shadow"
+    >
+      <div
+        class="search__lens search__lens--small search__bar--weight:focus"
+      ></div>
+      <input
+        class="search__bar search__bar--placeholder search___bar--small"
+        id="data-qa-npm small"
```

Comment:
The ID 'data-qa-npm small' contains a space, which is not valid for IDs. Consider using a hyphen or underscore instead.